inference-engine/src/vpu/myriad_plugin/myriad_config.cpp

```diff
@@ -33,19 +32,19 @@ IE_SUPPRESS_DEPRECATED_START
         CONFIG_KEY(DEVICE_ID),
 
         VPU_MYRIAD_CONFIG_KEY(FORCE_RESET),
+        ie::MYRIAD_FORCE_RESET,
```

Comment:
Is it better to separate old options that are here for compatibility reasons and going to be removed and new ones that are supposed  to be kept?

Reply:
Done